### PR TITLE
Update Plugin.swift

### PR DESCRIPTION
**Pull request by danyyacoub**
fix for swift 5

### CODE REVIEW

File: package.json
Comment (riderx) [inline package.json:45]: you need to remove this change to make allow merge

File: package.json
Comment (riderx) [inline package.json:48]: same here

File: package.json
Comment (riderx) [inline package.json:13]: same here

**Comment by squio** [comment]
Quick comment about this PR: I'm not the maintainer but I suppose the `dist` and all other extraneous files shouldn't be part of the VCS; just the two updated `getBool()` calls in the single swift file should be enough?

**Comment by riderx** [comment]
@squio  agree i do think he did that to allow npm i from github.
2 branch should exist, one to use before it's merge and one for the merge

**Comment by danyyacoub** [comment]
Yes exactly @riderx , i was trying to be able to make the npm install (you can see from the commits that it was a trial and error to make the npm i work :D )
I'll clean it up once i get a sense how to do it properly

**Comment by riderx** [comment]
@danyyacoub if you need help i can help :)

**Comment by danyyacoub** [comment]
Thank you @riderx , yes indeed, I actually took into account your note but still have several issues (the package is synced correctly but issues with the postinstall  build and syncing the plugins with android and ios), I used for reference the following 2 links 

https://blaipratdesaba.com/how-to-use-an-npm-node-module-that-has-been-forked-b7dd522fdd08

https://stackoverflow.com/questions/40528053/npm-install-and-build-of-forked-github-repo

I'm not using this fork for the time being but I'd love to have this experience and since this fork is already public i'd want to to make it clean and usable.

Thanks again

**Comment by riderx** [comment]
Hey @danyyacoub i do think the best way to make it publicly usable is to publish it in npm under your name :
https://jamescalmus.medium.com/how-to-publish-a-scoped-npm-package-for-your-organization-767af1c99b9f

that will result something like npm i @danyyacoub/capacitor-native-biometric` to use it.

when setting is done like in the tutorial you have to :
`npm run build & npm publish` and you are done :)

if you never create account in npm :
https://docs.npmjs.com/creating-a-new-npm-user-account

**Comment by adirery** [comment]
what's missing to get the merge?

**Comment by riderx** [comment]
@andrasnobank the PR include more than just the necessary changes, and there are no answers of @epicshaggy

**Comment by WuglyakBolgoink** [comment]
I'm not approve this PR because it contain a lot of unnecessary code changes. please close this PR without merge and create another one with dedicated changes related **ONLY** to Swift!!!

**Review by WuglyakBolgoink — CHANGES_REQUESTED** [review]
**NOT APPROVED**! see my last comment!

**Comment by riderx** [comment]
@WuglyakBolgoink @epicshaggy i made the PR with only what was necessary https://github.com/epicshaggy/capacitor-native-biometric/pull/53

**Comment by riderx** [comment]
PR can be close work have been done for it

**Pull request closed without merging**
